client/extensions/wp-super-cache/preload-tab.jsx

```diff
@@ -150,10 +150,10 @@ class PreloadTab extends Component {
 								onChange={ this.handlePreloadRefreshChange }>
 								<span>
 									{ translate(
-										'Refresh preloaded cache files every {{number /}} minute. ',
-										'Refresh preloaded cache files every {{number /}} minutes. ',
+										'Refresh preloaded cache files every {{number /}} minute ',
+										'Refresh preloaded cache files every {{number /}} minutes ',
 										{
-											count: preload_interval,
+											count: preload_interval || 0,
```

Comment:
I can't actually do that in this case. It will only set it to 0 if `preload_interval` is `undefined`, but it's actually `null`.